apps/web/src/pages/quick-start/steps/DigestPreview.tsx

```diff
@@ -93,6 +128,26 @@ function ThemeArrowLeft(props: React.ComponentPropsWithoutRef<'svg'>) {
   return <>{theme.colorScheme === 'dark' ? <ArrowLeft {...props} /> : <ArrowLeftGradient {...props} />}</>;
 }
 
-const StyledButton = styled(Button)`
+const StyledButton = styled(Button)<{ pulse: boolean }>`
   height: 50px;
+
+  ${({ pulse }) => {
+    return (
+      pulse &&
+      `
+      animation: pulse-animation 2s infinite;
+      @keyframes pulse-animation {
+        0% {
+          box-shadow: 0 0 0 0 rgba(255, 82, 82, 0.7);
+        }
+        70% {
+          box-shadow: 0 0 0 10px rgba(255, 82, 82, 0);
+        }
+        100% {
+          box-shadow: 0 0 0 0 rgba(255, 82, 82, 0);
+        }
+      }
+      `
+    );
+  }};
```

Comment:
I feel like we should add it to the Button from the `design-system` folder

Reply:
we most definitely can i stopped to add to design-system because I'm afraid abuse it

Reply:
for this case it's fine